README.md

```diff
@@ -362,6 +363,25 @@ intArrayPublisher.send([10, 2, 2, 4, 3, 8])
 ["10", "2", "2", "4", "3", "8"]
 ```
 
+### Distinct
+
+`Publisher.distinct` and `.distinct(by:)` are stricter forms of Apple’s [`Publisher.removeDuplicates`](https://developer.apple.com/documentation/combine/publisher/3204745-removeduplicates) and [`.removeDuplicates(by:)`](https://developer.apple.com/documentation/combine/publisher/3204746-removeduplicates)—the operators de-duplicate across _all_ previous value events, instead of pairwise.
```

Comment:
The biggest issue I have is still with the name - `distinct` operators are from Rx, where here we might want to keep the same naming convention. My currrent instinct is something like `removeUniqueDuplicates`, `removeAllDuplicates` or `removeDuplicatesUniquely` but I'm very open to other variations, preferably coming from the same family.

Reply:
Game for `removeAllDuplicates`, I’ll rename everything `distinct`-related in an upcoming commit. 😎